examples/molnet/train_molnet.py

```diff
@@ -0,0 +1,205 @@
+import argparse
+import os
+
+import chainer
+from chainer import iterators
+from chainer import optimizers
+from chainer import training
+from chainer.training import extensions as E
+import numpy
+
+from chainer_chemistry import datasets as D
+from chainer_chemistry.models import MLP, NFP, GGNN, SchNet, WeaveNet, RSGCN  # NOQA
+
+from chainer_chemistry.models.prediction import Classifier
+from chainer_chemistry.models.prediction import Regressor
+from chainer_chemistry.dataset.converters import concat_mols
+from chainer_chemistry.dataset.preprocessors import preprocess_method_dict
+from chainer_chemistry.datasets import NumpyTupleDataset
+from chainer_chemistry.datasets.molnet.molnet_config import molnet_default_config # NOQA
+
+
```

Comment:
Now, I fell we can put it in the library code... (maybe in next PR).

Reply:
Is this about `GraphConvPredictor` ?

Reply:
yes